auto_nag/scripts/assignee_no_login.py

```diff
@@ -69,6 +69,9 @@ def get_bz_params(self, date):
             "f1": "assignee_last_login",
             "o1": "lessthan",
             "v1": start_date,
+            "f2": "days_elapsed",
+            "o2": "lessthan",
+            "v2": 90,
```

Comment:
It should be added in the conf file instead.

Reply:
Done